gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/vertx/web/auth/handler/impl/OAuth2AuthHandlerImpl.java

```diff
@@ -100,36 +100,31 @@ public void handle(RoutingContext context) {
                 // check if current subject can access its own resources
                 if (selfResource) {
                     final String resourceId = context.request().getParam(resourceParameter);
```

Comment:
should we not extract method of this large method?

Reply:
It has a less than 15 complexity.